src/main/resources/templates/db_scripts/schema.json

```diff
@@ -108,12 +108,17 @@
     {
       "run": "after",
       "snippetPath": "migration_scripts/change_progress_total_data_type.sql",
-      "fromModuleVersion": "mod-data-export-3.1.0"
+      "fromModuleVersion": "mod-data-export-3.0.1"
```

Comment:
This was part of https://issues.folio.org/browse/MDEXP-292. There is no version marked for it, but since it is not a bug, I think we will not release it as part of this release ? 
Can you please cross-check that

Reply:
ask Magda do we need this ticket in the release, thanks